From 89ab141da2df07d793135b31037797b2f34e5be9 Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Tue, 19 Oct 2021 14:40:28 +0200
Subject: [PATCH] IWYU: missing include for numeric limits used in partition alloc constants.

Fix build:
    ../../base/allocator/partition_allocator/partition_alloc_constants.h:355:43: error: ‘numeric_limits’ is not a member of ‘std’
      355 | constexpr size_t kMaxFreeableSpans = std::numeric_limits<int8_t>::max();
          |                                           ^~~~~~~~~~~~~~
    ../../base/allocator/partition_allocator/partition_alloc_constants.h:355:64: error: expected primary-expression before ‘>’ token
      355 | constexpr size_t kMaxFreeableSpans = std::numeric_limits<int8_t>::max();
          |                                                                ^
    ../../base/allocator/partition_allocator/partition_alloc_constants.h:355:67: error: ‘::max’ has not been declared; did you mean ‘std::max’?
      355 | constexpr size_t kMaxFreeableSpans = std::numeric_limits<int8_t>::max();
          |                                                                   ^~~
          |                                                                   std::max

Bug: 957519
Change-Id: I0a2f839589650901c5abf1feae5f3a3ef18fefc6
---

diff --git a/base/allocator/partition_allocator/partition_alloc_constants.h b/base/allocator/partition_allocator/partition_alloc_constants.h
index b130900..cdcf87b 100644
--- a/base/allocator/partition_allocator/partition_alloc_constants.h
+++ b/base/allocator/partition_allocator/partition_alloc_constants.h
@@ -9,6 +9,7 @@
 #include <cstddef>
 
 #include <algorithm>
+#include <limits>
 
 #include "base/allocator/partition_allocator/address_pool_manager_types.h"
 #include "base/allocator/partition_allocator/page_allocator_constants.h"
